recipes/itk/all/patches/CMakeProjectWrapper.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.0)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(WIN32)
```

Comment:
This looks like an error of some kind to me. This should definitely not be necessary

Reply:
done